src/main/java/io/nats/client/NUID.java

```diff
@@ -98,15 +99,15 @@ public NUID() {
      * @return the next NUID string from a shared global NUID instance
      */
     public static synchronized String nextGlobal() {
-        return getInstance().next();
+        return StandardCharsets.US_ASCII.decode(getInstance().next()).toString();
     }
 
     /**
      * Generate the next NUID string from this instance.
      *
      * @return the next NUID string from this instance.
      */
-    public final synchronized String next() {
+    public final synchronized ByteBuffer next() {
```

Comment:
This could break existing users... perhaps deprecate, and add `nextByteBuffer()`?

Reply:
Changed back to return a `String`.